apps/web/components/Sponsor/Information.tsx

```diff
@@ -27,7 +27,7 @@ function SponsorInformation({
       <div className="flex flex-wrap gap-5 md:gap-20 justify-center">
         <Button
           disabled={isExpired}
-          className="mt-3 mb-10"
+          className="w-full sm:w-auto mt-0 sm:mt-3 md:mb-10 sm:mb-3 mb-0"
```

Comment:
[IMO] 
ご対応ありがとうございます！

余白の設定ですが、各ボタンごとに margin を 適用するのではなく、`gap`  や ボタンを囲む div タグに margin を利用するといいと思います！

記述量が減るので保守性があがるため、このように修正していただけると！

```suggestion
          className="w-full sm:w-auto"
```